flink-table/flink-table-common/src/test/java/org/apache/flink/table/expressions/ExpressionTest.java

```diff
@@ -281,6 +281,12 @@ public void testPeriodValueLiteralExtraction() {
 	// --------------------------------------------------------------------------------------------
 
 	private static Expression createExpressionTree(Integer nestedValue) {
+		final ValueLiteralExpression nestedLiteral;
+		if (nestedValue != null) {
+			nestedLiteral = new ValueLiteralExpression(nestedValue, DataTypes.INT().notNull());
```

Comment:
nit: add a `@Nullable` annotation to `ValueLiteralExpression` constructor parameters